src/common/components/forms/InputOutline.tsx

```diff
@@ -0,0 +1,31 @@
+import { FormControl, Input, InputProps, VStack } from '@chakra-ui/react';
+import * as React from 'react';
+import { Controller, Control } from 'react-hook-form';
+import { FormErrorMessage } from '@common/components';
+
+type CustomerInputOutlineProps = {
+  control: Control<any, any>;
+  name: string;
+} & InputProps;
+export default function CustomInputOutline({
```

Comment:
Same, see if you can merge this with `CustomInput`

Reply:
this has been updated